src/Features/Core/Portable/SemanticClassificationCache/SemanticClassificationCacheService.cs

```diff
@@ -7,41 +7,29 @@
 using System.Composition;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Classification;
-using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
 using Microsoft.CodeAnalysis.Host.Mef;
-using Microsoft.CodeAnalysis.PersistentStorage;
 using Microsoft.CodeAnalysis.PooledObjects;
 using Microsoft.CodeAnalysis.Remote;
 using Microsoft.CodeAnalysis.Text;
-using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem;
 
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.SemanticClassificationCache
+namespace Microsoft.CodeAnalysis.SemanticClassificationCache
 {
-    [ExportWorkspaceService(typeof(ISemanticClassificationCacheService), ServiceLayer.Host), Shared]
-    internal class VisualStudioSemanticClassificationCacheService
-        : ForegroundThreadAffinitizedObject, ISemanticClassificationCacheService
+    [ExportWorkspaceService(typeof(ISemanticClassificationCacheService), ServiceLayer.Editor), Shared]
```

Comment:
So this should be Editor layer now that it's in Features.

Reply:
It's either supposed to be `Desktop` or `Editor`. The latter seemed most reasonable to me?

Reply:
Desktop is legacy, should be removed. What's the meaning of Editor?

Reply:
I think it would be more clear in the future if we did a refactoring to [ServiceLayer](https://github.com/dotnet/roslyn/blob/main/src/Workspaces/Core/Portable/Workspace/Host/Mef/ServiceLayer.cs) so that the names matched the terms we actually use to talk about these layers.